recommendations/test_sms_parser.py

```diff
@@ -9,10 +9,6 @@ def setUp(self):
         self.faker = Faker()
         self.phone = self.faker.phone_number()
 
-    def setUp(self):
-        self.faker = Faker()
-        self.phone = self.faker.phone_number()
-
```

Comment:
Don't know why we included the `setUp` method twice? Probably an error when we merged our testing branches in.

Reply:
That is weird. I didn't notice.